repository-data/application/src/main/resources/hcm-config/namespaces/hee/publicationLandingPage.yaml

```diff
@@ -151,16 +127,23 @@ definitions:
             hipposysedit:primary: false
             hipposysedit:type: String
             hipposysedit:validators: [email, required]
+          /publicationTypeClassifiable:
```

Comment:
Yes, will look to implement that

Reply:
It can't be achieved with a field level validator, so I've created a new document validation routine and that will pick up if the value has not been supplied.